core/domain/exp_domain.py

```diff
@@ -2905,9 +3833,9 @@ def to_metadata_dict(self):
         Returns:
             A metadata dict for the given exploration summary.
             The metadata dict has three keys:
-                'id': the exploration id
-                'title': the exploration title
-                'objective': the exploration objective
+                'id': the exploration id;
```

Comment:
I think you don't need the semicolons at the ends of the lines. We don't use them anywhere else. You could use hyphens (with a space after them) before the lines instead.

Also, consider documenting as follows:
```
    The metadata dict has three keys:
    - 'id': str. The exploration ID.
    - 'title': str. The exploration title.
    - 'objective': str. The exploration objective.
```

Note how the type information is specified. This should be done for other dicts as well.